### PR TITLE
lockdown: add an option to delete different pairs

### PR DESCRIPTION
**Pull request by yotamolenik**
I think we want to leave the cli command as is

### CODE REVIEW

File: pymobiledevice3/lockdown.py [hunk 1]
```diff
@@ -441,8 +441,13 @@ def pair(self, timeout: int = None) -> None:
         self.paired = True
 
     @reconnect_on_remote_close
-    def unpair(self) -> Mapping:
-        return self._request('Unpair', {'PairRecord': self.pair_record, 'ProtocolVersion': '2'}, verify_request=False)
+    def unpair(self, host_id: str = None) -> Mapping:
+        if host_id is not None:
+            pair_record = {'Host-ID': host_id}
```
Comment (doronz88) [inline pymobiledevice3/lockdown.py]: - `HostID`
- was it tested?

**Review by doronz88 — CHANGES_REQUESTED** [review]
make the commit message clearer:

```
lockdown: add `host_id` option to `unpair()`
```

File: pymobiledevice3/lockdown.py [hunk 2]
```diff
@@ -140,6 +140,7 @@ def __init__(self, serial: str = None, hostname: str = None, client_name: str =
         self.paired = False
         self.session_id = None
         self.host_id = self.generate_host_id(local_hostname)
+        print(self.host_id)
```
Comment (doronz88) [inline pymobiledevice3/lockdown.py]: remove

File: pymobiledevice3/lockdown.py [hunk 3]
```diff
@@ -441,8 +442,13 @@ def pair(self, timeout: int = None) -> None:
         self.paired = True
 
     @reconnect_on_remote_close
-    def unpair(self) -> Mapping:
-        return self._request('Unpair', {'PairRecord': self.pair_record, 'ProtocolVersion': '2'}, verify_request=False)
+    def unpair(self, host_id: str = None) -> Mapping:
```
Comment (doronz88) [inline pymobiledevice3/lockdown.py]: - change to `-> None`
- don't return, simply request it

**Review by doronz88 — CHANGES_REQUESTED** [review]

File: pymobiledevice3/lockdown.py (hunk 3 above)
Comment (yotamolenik) [reply pymobiledevice3/lockdown.py]: done

File: pymobiledevice3/lockdown.py (hunk 2 above)
Comment (yotamolenik) [reply pymobiledevice3/lockdown.py]: done

**Pull request merged**